client/k8srestconfig/k8s_rest_config.go

```diff
@@ -111,19 +111,19 @@ func New(config Config) (*rest.Config, error) {
 
 	// Settings.
 	if config.Address == "" && !config.InCluster && config.KubeConfig == "" {
-		return nil, microerror.Maskf(invalidConfigError, "%T.Address must not be empty when not using %T.InCluster or %T.KubeConfig", config)
+		return nil, microerror.Maskf(invalidConfigError, "%T.Address must not be empty when not using %T.InCluster or %T.KubeConfig", config, config, config)
```

Comment:
It would make sense to format the string using `fmt.Sprintf` and position argument

```suggestion
		return nil, microerror.Maskf(invalidConfigError, fmt.Sprintf("%[1]T.Address must not be empty when not using %[1]T.InCluster or %[1]T.KubeConfig", config))
```

Reply:
If you approve this change you need to add "fmt" to import.

Reply:
I think we agreed a while ago that we don't want to use `%[n]` in our string formats. Also even we don't need fmt.Sprintf because Maskf calls it under the hood anyway. I'll go with the current version.